metadata.json

```diff
@@ -45,6 +45,10 @@
     {
       "name": "richardc/datacat",
       "version_requirement": ">= 0.6.0 < 1.0.0"
+    },
+    {
+      "name": "puppet/mosquitto",
+      "version_requirement": ">= 1.0.0 < 2.0.0"
```

Comment:
Can you make this >= 1.1.0 please? It contains fixes we need.

Reply:
done.